docs/tutorial.md

```diff
@@ -91,31 +90,30 @@ NAME
 echo
 ```
 
-The status of type `Succeeded` shows that the `Task` ran successfully.
+The `Succeeded` status confirms that the `Task` completed with no errors.
+
+To see more detail about the execution of your `Task` view its logs:
 
-To see the actual outcome, use the following command:
 ```bash
 tkn taskrun logs echo-hello-world-task-run
 ```
 
-You will get an output similar to this:
+Below is the example output:
 
 ```
 [echo] hello world
 ```
 
-### Task Inputs and Outputs
+### Specifying the inputs and outputs for your `Task`
```

Comment:
I prefer the original title here. Was there a reason you felt this is not clear?

Reply:
Same as before. You want to start with an action to indicate you're about to complete a procedure. "Task Inputs and Outputs" is a little vague in the context of the tutorial's flow. You also want to be consistent with the rest of the headings in the document.

Reply:
Maybe then `Specifying Task Inputs and Outputs`?

Reply:
Sure, that's cleaner than my version.